outbound/index.js

```diff
@@ -220,14 +220,14 @@ exports.send_trans_email = function (transaction, next) {
     // add potentially missing headers
     if (!transaction.header.get_all('Message-Id').length) {
         logger.loginfo("[outbound] Adding missing Message-Id header");
-        transaction.add_header('Message-Id', '<' + transaction.uuid + '@' + config.get('me') + '>');
+        transaction.add_header(`Message-Id<${transaction.uuid}@${config.get('me')}>`);
```

Comment:
oops! The call signature for add_header requires two arguments. It needs to look like this instead:
```js
transaction.add_header('Message-Id', `<${transaction.uuid}@${config.get('me')}>`);
```

Reply:
Ah, I overlooked the comma. I'll push the fix in a bit.